meta-mender-nxp/README.md

```diff
@@ -25,3 +25,36 @@ Setup environment
 Build
 
     $ bitbake core-image-base
+
+
+## Dependencies
```

Comment:
Move this section before the 'Quick Start` section.

Reply:
Updated